docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -165,15 +169,49 @@ To discard your proposal after confirming the validator was added, call `ibft_di
     ```bash
     curl -X POST --data '{"jsonrpc":"2.0","method":"ibft_discardValidatorVote","params":["0xFE3B557E8Fb62b89F4916B721be55cEb828dBd73"], "id":1}' <JSON-RPC-endpoint:port>
     ```
-### Removing a Validator
+#### Removing a Validator
 
 The process for removing a validator is the same as adding a validator except you specify `false` as the second parameter of `ibft_proposeValidatorVote`.
 
-### Epoch Transition
+#### Epoch Transition
 
 At each epoch transition, all pending votes collected from received blocks are discarded. Existing proposals remain 
 in effect and validators re-add their vote the next time they create a block. 
 
 An epoch transition occurs every `epochLength` blocks where `epochlength` is defined in the IBFT genesis file.
 
+### Adding and Removing Validators without Voting
+
+In some cases, such as when the results of voting would not yield a majority, you can bypass voting and manually change validators in your genesis file.
+
+To add or remove validators without voting:
+
+1. Stop the network.
+1. Copy the `transitions` object and paste it after the `config` object in your genesis file. Specify your own values, where
+
+    * `<BlockNumber>` is the upcoming block where you want the change in validators to occur.
+    * `<AValidatorAddress>` is the account address for a validator to be added.
```

Comment:
Why the initial A in Validator address ?

Reply:
Updated.